static/js/vue-cdr-access/src/components/full_record/restrictedContent.vue

```diff
@@ -13,12 +13,14 @@
             <a class="edit button action" :href="editDescriptionUrl(recordData.briefObject.id)"><i class="fa fa-edit"></i> {{ $t('full_record.edit') }}</a>
         </div>
         <template v-if="recordData.resourceType === 'File'">
-            <template v-if="hasPermission(recordData, 'viewOriginal')">
+            <template v-if="hasDownloadAccess(recordData)">
                 <div class="header-button" v-html="downloadButtonHtml(recordData.briefObject)"></div>
-                <div class="actionlink">
-                    <a class="button view action" :href="recordData.dataFileUrl">
-                        <i class="fa fa-search" aria-hidden="true"></i> View</a>
-                </div>
+                <template v-if="hasPermission(recordData, 'viewOriginal')">
```

Comment:
You should be able to put the `v-if` on `<div class="actionlink">`. I'm not sure why I did it this way, but it's not a big deal either way.